INSTALL

```diff
@@ -157,6 +157,12 @@ any of the following arguments (not a definitive list) to 'configure':
     released in bulk, thus reducing the total number of mutex operations.  See
     the "opt.tcache" option for usage details.
 
+--disable-thp
+    Disable transparent huge page (THP) integration.  On systems with THP
+    support, THPs are explicitly avoided for chunks that back small and/or large
+    allocations, because such chunks typically comprise a active, inactive
```

Comment:
nit: The "a" before active is extra? Also does this mean we explicitly disable thp unless its huge allocation?